js/render/draw_background.js

```diff
@@ -60,7 +58,7 @@ function drawBackground(painter, source, layer) {
     // the depth and stencil buffers get into a bad state.
     // This can be refactored into a single draw call once earcut lands and
     // we don't have so much going on in the stencil buffer.
-    var coords = pyramid.coveringTiles(transform);
+    var coords = transform.coveringTiles({ tileSize: 512 });
     for (var c = 0; c < coords.length; c++) {
         var coord = coords[c];
         var tileSize = 512;
```

Comment:
Seems like we're using `512` as a magic number in a few places. Could you factor this out into a constant?